engine/src/test/java/com/datatorrent/stram/client/StramClientUtilsTest.java

```diff
@@ -175,6 +179,9 @@ public void getApexDFSRootDirLegacy() throws IOException
     FileSystem fs = FileSystem.newInstance(conf);
     Path path = StramClientUtils.getApexDFSRootDir(fs, conf);
     Assert.assertEquals("file:/a/b/c", path.toString());
+    // verify conf properties for DT_DFS_ROOT_DIR and APEX_APP_DFS_ROOT_DIR
```

Comment:
Why is YarnConfiguration necessary? It is cast to Configuration anyway.

Reply:
Good point. I copied the pattern from an existing test (most probably `testRMWebAddress()` where it is actually required). But I will change it to Configuration and verify.

Reply:
fixed